cmd/logger.go

```diff
@@ -154,36 +150,45 @@ func logIf(level logrus.Level, source string, err error, msg string, data ...int
 	if err == nil || isErrIgnored(err) {
 		return
 	}
-
-	fields := logrus.Fields{
-		"source": source,
-		"cause":  err.Error(),
-	}
-
-	if terr, ok := err.(*errors.Error); ok {
-		fields["stack"] = strings.Join(terr.Stack(), " ")
+	var trace []string
+	cause := err.Error()
+	// Get trace trace
+	trace = getTrace(3)
```

Comment:
No need to declare `trace`. you could use `:=` here

Reply:
Done.